mlflow/pytorch/torchserve/tests/test_cli.py

```diff
@@ -0,0 +1,137 @@
+import atexit
+import json
+import os
+import pytest
+import shutil
+import subprocess
+import time
+from click.testing import CliRunner
+
+from mlflow import deployments
+from mlflow.deployments import cli
+
+f_target = "torchserve"
+f_deployment_id = "test"
+f_flavor = None
+f_model_uri = "mlflow/pytorch/torchserve/tests/resources/linear.pt"
```

Comment:
Maybe use `os.path.join` to construct these paths? Otherwise these tests won't be able to run against Windows, if we eventually want to test the plugin against Windows